maven_iuvs/instrument.py

```diff
@@ -239,3 +243,67 @@ def mcp_volt_to_gain(volt, channel="FUV"):
         gain = A*np.exp(alpha*volt)
 
     return gain
+
+
+# TODO: Move these to appropriate places once done.
+def convert_spectrum_DN_to_photons(light_fits, spectrum):
```

Comment:
I think this should be called `convert_spectrum_DN_to_photoevents` and a note added in the description that this does NOT fully calibrate the data.

Reply:
Yeah, good catch. The rest of the function references photoevents, so the name must have been an artifact.